packages/docs/config/introduction.md

```diff
@@ -41,3 +40,32 @@ export default defineConfig({
   // ...
 })
 ```
+
+## Example config 
+
+[CB Insights](https://www.cbinsights.com/) heavily customizes the config for their use case:
+
+```js
+import { defineConfig } from '@figus/cli';
+import dotenv from 'dotenv';
+import { pascalCase } from 'change-case';
+
+dotenv.config();
+
+export default defineConfig({
+  getFileName: (svgPathObj) => {
+    return `${pascalCase(svgPathObj.name)}Icon.tsx`;
+  },
+  output: 'packages/cbicons/src/components',
+  template: 'packages/cbicons/src/icon.mustache',
+  framework: 'react',
+  getComponentName: (name) => `${name}Icon`,
+  path: './packages/cbicons/src/assets',
+  figma: {
+    fileKey: 'qEm8vdvuoqzn51sx1JOngS',
+    // Create a .env file and put your token there
+    token: process.env.FIGMA_API_TOKEN,
```

Comment:
A lot of developers are going to need to add a token. Giving them an example of how to do so without committing it, will help them.